res/controllers/Reloop-Digital-Jockey-2-IE.midi.xml

```diff
@@ -0,0 +1,1243 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.1.0+" schemaVersion="1">
+    <info>
+        <name>Reloop Digital Jockey 2 Interface Edition</name>
+        <author>DJ aK</author>
+        <description>Controller mapping for Reloop Digital Jockey 2 Interface Edition</description>
+        <forums>https://mixxx.discourse.group/t/reloop-digital-jockey-2-mapping-by-dj-ak/23971</forums>
+        <manual>reloop_digital_jockey_2_interface_edition</manual>
+    </info>
+    <controller id="DIGITALJOCKEY2">
+        <scriptfiles>
+            <file functionprefix="" filename="lodash.mixxx.js"/>
+            <file functionprefix="" filename="midi-components-0.0.js"/>
+            <file functionprefix="RDJ2" filename="Reloop-Digital-Jockey-2-IE.scripts.js"/>
+        </scriptfiles>
+
+        <controls>
+        <!-- Master -->
+            <control>
+                <group>[Master]</group>
+                <key>gain</key>
+                <status>0xB0</status>
+                <midino>0x0F</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <group>[Master]</group>
+                <key>headGain</key>
+                <status>0xB0</status>
+                <midino>0x10</midino>
+                <options>

```

Comment:
You should be able to remove also those lines (repeat the process for each occurence), since I see no xsd file validating that it must be present.
But I'm not sure, if Mixxx needs it for whatever reason. So please test it first with one. And if not, you can remove all occurences of that.

Reply:
I would appreciate confirmed remarks. I have limited time for modifications and testing based on someone's doubts so if something works and doesn't have negative impact on anything, I would keep it as it is.

Reply:
Let me do it, schould be much faster

Reply:
I had to revert your commit. It broke the mapping. Those lines were necessary.